Source/AtomicEditor/Editors/SceneEditor3D/SceneEditor3D.cpp

```diff
@@ -259,6 +282,8 @@ void SceneEditor3D::Close(bool navigateToAvailableResource)
         sceneImporter_ = nullptr;
     }
 
+    debugHud_->SetSceneOpen(false);
+
```

Comment:
This is the only time SetSceneOpen is called with false?  Though the scene is still open?  How does this play with multiple scenes being opened at same time?